doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1654,30 +1702,31 @@ Defining |Ltac| symbols
    .. prodn::
       tacdef_body ::= @qualid {* @name } {| := | ::= } @ltac_expr
 
-   Defines or redefines an |Ltac| symbol that can be used in any tactic
-   script or |Ltac| toplevel definition.
+   Defines or redefines an |Ltac| symbol.
 
-   If the :attr:`local` attribute is specified, the tactic definition will not be
+   If the :attr:`local` attribute is specified, the definition will not be
    exported outside the current module.
 
    :token:`qualid`
-      Name of the tactic being defined
+      Name of the symbol being defined or redefined
 
    :n:`{* @name }`
-      If one or more :token:`name`\s are specified, they are the parameters of
-      the tactic being defined.  If no names are specified, :token:`qualid` is
-      assigned the value of :token:`ltac_expr`.
+      If specified, the symbol defines a function with the given parameter names.
+      If no names are specified, :token:`qualid` is assigned the value of :token:`ltac_expr`.
 
    `:=`
-      Defines a user-defined tactic, but gives an error if a tactic
-      has already been defined with the specified name.
+      Defines a user-defined symbol, but gives an error if the symbol has already
+      been defined.
+
+   .. todo inconsistency: "Ltac intros := idtac" redefines/hides an existing tactic
+      You can't use ::=
 
       .. exn:: There is already an Ltac named @qualid
          :undocumented:
 
    `::=`
-      Redefines an existing user-defined tactic, but gives an error if no
-      tactic has been defined with the specified name.  The redefinition
+      Redefines an existing user-defined symbol, but gives an error if the
+      symbol doesn't exist.  If :attr:`local` is not specified, the redefinition
```

Comment:
I liked the version with `tactic` better, but I'm also okay with the `symbol` version.
```suggestion
      Redefines an existing user-defined tactic, but gives an error if no
      tactic has been defined with the specified name.  Note that :cmd:`Tactic Notation`\s
      do not count as user-defined tactics for `::=`.  If :attr:`local` is not specified, the redefinition
```

Reply:
Well, in `Ltac n ::= 2`, n is not a tactic.

Reply:
Ah, okay.  I'm a bit concerned that "user-defined symbol" might make it sound like we can redefine things like `Definition n := 1` with `Ltac n ::= 2`, which is not the case.

Reply:
Perhaps, but they would figure that out pretty quickly.  The context for this text is Ltac definitions, not all of Coq, so thinking it refers to `Definition`s would be a little odd.